integrations/chroma/src/haystack_integrations/document_stores/chroma/document_store.py

```diff
@@ -45,22 +46,41 @@ def __init__(
         :param collection_name: the name of the collection to use in the database.
         :param embedding_function: the name of the embedding function to use to embed the query
         :param persist_path: where to store the database. If None, the database will be `in-memory`.
+        :param distance_function: distance metric for the embedding space. Valid options are
+        'l2', 'cosine' and 'ip'.
```

Comment:
Good idea to use Enum here for validation.

Reply:
Literal should also be fine.

In case you want to introduce an Enum, I recommend looking for similar implementations in our repositories: for example https://github.com/deepset-ai/haystack/blob/fd1a06d17173e7bd5488c1540f6a5dd70a14486a/haystack/utils/hf.py#L26
We can allow users pass an Enum or a str and perform automatic validation using `from_str`.

Reply:
Thanks!! I implemented a similar `Enum` functionality but it feels like an overkill here. So I am going back to `Literal`  😄